frontend/pages/admin/charities/index.tsx

```diff
@@ -27,7 +65,32 @@ const AdminCharities = () => {
             Create
           </Button>
         </Stack>
+
+        <SimpleTable
+          rows={charities ?? []}
+          isLoading={!charities}
+          columns={[
+            { title: 'ID', key: 'id' },
+            { title: 'Logo', key: 'logoBase64', transformValue: createLogos },
+            { title: 'Name', key: 'name' },
+          ]}
+          actions={[deleteCharityAction, editCharityAction]}
+        />
       </Box>
+
+      {selectedCharity && (
```

Comment:
Seems like we don't need this as the dialog only opens based on the `open` prop
```suggestion
      {(
```